### PR TITLE
Add script to post-process aggregates

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This adds a breaking change to the `prio publish` command by changing the output schema from an array to an object. The `prio-processor fetch-origin` job creates an array that can be zipped with the resulting publish payload to create a tabular json file for insertion into BigQuery.

### CODE REVIEW

File: processor/prio_processor/bootstrap.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def run(output, credentials):
 
     logging.info(f"writing runner.py to {output}/runner.py")
     with fs.open(f"{output}/runner.py", "w") as f:
-        f.write("from prio_processor import staging; staging.run()")
+        f.write(f"from prio_processor import cli; cli.entry_point()")
```
Comment (wlach) [inline processor/prio_processor/bootstrap.py]: I don't think this needs to be a format string?

File: processor/prio_processor/indexing.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    def _apply_structure(batch_id, payload):
+        """Create a user-defined function that maps partitioned batch-ids into
+        list of structures containing the aggregate value and its metadata."""
+
+        # assumption: hyphens are used to define a partition of origins
+        if batch_id not in config:
+            return []
+
+        # currently all batch-ids contain a single hyphen with 2 parts
+        split = batch_id.split("-")
```
Comment (wlach) [inline processor/prio_processor/indexing.py]: Should we assert on this condition?

File: processor/bin/insert [hunk 3]
```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
```
Comment (wlach) [inline processor/bin/insert]: Maybe add a one liner comment on what this script is for?

File: processor/prio_processor/indexing.py (hunk 2 above)
Comment (acmiyaguchi) [reply processor/prio_processor/indexing.py]: Sure, it seems reasonable -- otherwise it should raise an IndexError on line 61.

**Pull request merged**